### PR TITLE
Issue #16 - Remove "Show" interface. Always be in "edit" mode for data fields in a record.

### PR DESCRIPTION
**Pull request by MrJadaml**
I have changed it so that users will only ever be in edit mode. I have also added color coded flash messages so that it more apparent when they update items as many times they will be re-rendered in the same view.

### CODE REVIEW

File: app/controllers/patients_controller.rb [hunk 1]
```diff
@@ -23,13 +19,22 @@ def edit
 
   def create
     @patient = Patient.new(patient_params)
-    @patient.save
-    respond_with(@patient)
+    if @patient.save
+      flash[:success] = "You have successfully created a patient."
+      render :edit
+    else
+      flash[:error] = "Something went creating this patient. Please try again."
```
Comment (awong-dev) [inline app/controllers/patients_controller.rb]: nit: "went creating" -> "went wrong"

File: app/helpers/patients_helper.rb [hunk 2]
```diff
@@ -1,2 +1,17 @@
 module PatientsHelper
+
+  def bootstrap_class_for flash_type
+    { success: 'alert-success', error: 'alert-danger', warning: 'alert-warning'}[flash_type.to_sym]
```
Comment (awong-dev) [inline app/helpers/patients_helper.rb]: do we need the to_sym here? I would have expected the string to have just worked...

Also, a case statement seems clearer.

File: app/helpers/patients_helper.rb [hunk 3]
```diff
@@ -1,2 +1,17 @@
 module PatientsHelper
+
+  def bootstrap_class_for flash_type
```
Comment (awong-dev) [inline app/helpers/patients_helper.rb]: Use parens for function definitions?

  def bootstrap_class_for(flash_type)

seems to better follow the ruby style guide... https://github.com/bbatsov/ruby-style-guide

File: app/helpers/patients_helper.rb (hunk 2 above)
Comment (MrJadaml) [inline app/helpers/patients_helper.rb]: It initially was just a string but that was not picking it up after which I switched it over `.to_sym` and it worked.

File: app/helpers/patients_helper.rb (hunk 3 above)
Comment (MrJadaml) [inline app/helpers/patients_helper.rb]: Changed that one.

File: app/controllers/patients_controller.rb (hunk 1 above)
Comment (MrJadaml) [inline app/controllers/patients_controller.rb]: changed to: "Something went wrong when creating this patient. Please try again."

**Comment by MrJadaml** [comment]
The edit view did not have that list of buttons (to be sidebar nav) I am currently adding those into the edit view and changing it to that sidebar nav which will also knock out issue #36

**Comment by MrJadaml** [comment]
Updated PR via notes. I also threw in an horizontal reference <hr> on the edit page. I think helps in separating the header from the forms/nav let me know if you like that or not. If so we can toss them into the other pages as well.

**Comment by awong-dev** [comment]
Thanks! I'll merge this in.

FYI I was walking down the path of creating a nested layout with the side-nav so that all the major view modes could have such a nav. Add patient, for example, makes more sense to me as a method on the registry page rather than as a top-nav.

**Pull request merged**